tests/explainers/test_tree.py

```diff
@@ -144,30 +144,49 @@ def test_xgboost_direct():
 
 
 def test_xgboost_multiclass():
-    xgboost = pytest.importorskip('xgboost')
+    xgboost = pytest.importorskip("xgboost")
 
     # train XGBoost model
     X, y = shap.datasets.iris()
-    model = xgboost.XGBClassifier(n_estimators=1, max_depth=4)
+    model = xgboost.XGBClassifier(n_estimators=10, max_depth=4)
     model.fit(X, y)
+    predicted = model.predict(X, output_margin=True)
 
+    # explain the model's predictions using SHAP values
     explainer = shap.TreeExplainer(model)
 
-    assert np.allclose(explainer.model.predict(X), model.predict(X, output_margin=True))
+    assert np.allclose(explainer.model.predict(X), predicted)
 
-    shap_values = explainer.shap_values(X)
+    explanation = explainer(X)
+    # check the properties of Explanation object
+    num_classes = 3
+    assert explanation.values.shape == (*X.shape, num_classes)
+    assert explanation.base_values.shape == (len(X), num_classes)
```

Comment:
The general crux of all of the changes in this `test_tree.py` file is to utilize the `explainer.__call__(X)` instead of `explainer.shap_values(X)`, and explicitly asserting the shape of the base values and SHAP values arrays.